### PR TITLE
Timers 4 each eva situation

### PR DESCRIPTION
**Pull request by adyxn**
Displayed station Timers and added comments

### CODE REVIEW

File: client/components/ui/timing.tsx [hunk 1]
```diff
@@ -59,22 +60,180 @@ const MissionStopwatch = () => {
...
+    return () => clearInterval(interval);
+}, [/* dependencies */]);
+
+    return (
+   
+     
       <div className="flex flex-col">
-        <p className="font-bold text-md">Current Time</p>
-        <MissionClock />
+          <div className="text-1xl font-semibold mb-1 absolute pb-0">
```
Comment (abhi-arya1) [inline client/components/ui/timing.tsx:87]: Make text equal to `text-2xl`

**Review by abhi-arya1 — CHANGES_REQUESTED** [review]
- Reduce top padding between the mission/current time and these timers 
- Reduce padding between each timer object
- If you can make them as a grid, i.e:
UIA Spec Rover DCU
(you can do this with this structure:
```html
<div className="flex flex-row">
      put all 4 timers here
</div>
```
which will make that grid for you)
- Contact @abhi-arya1 with questions

**Review by Turfader — CHANGES_REQUESTED** [review]
See Abhi's comments

**Review by abhi-arya1 — APPROVED** [review]
lgtm--fixed in code review
2/3/24

**Review by Turfader — APPROVED** [review]
fixed in code review

**Pull request merged**